<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>radeon_profile</class>
 <widget class="QMainWindow" name="radeon_profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>218</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>292</width>
    <height>174</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>218</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Radeon Profile</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>85</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current GPU clock:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current mem clock:</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentGPU">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>800 MHz</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="currentMEM">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>600 MHz</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Voltage:</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentVolts">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>900 mV</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="stdProfiles">
     <attribute name="title">
      <string>Standard profiles</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>63</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Current profile:</string>
       </property>
      </widget>
      <widget class="QLabel" name="currentProfile">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>9</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>default</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="chProfile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>36</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Change profile</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="dpmProfiles">
     <attribute name="title">
      <string>DPM profiles</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>63</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Current profile:</string>
       </property>
      </widget>
      <widget class="QLabel" name="currentDpmProfile">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>performance</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_chDpmProfile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>36</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Change profile</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
